src/SearchResults.JS

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import Moment from "moment";
+function SearchResult({results}) {
+    return <div>
+        <table class="table">
+  <thead>
+    <tr>
+      <th scope="col">id</th>
+      <th scope="col">title</th>
+      <th scope="col">firstName</th>
+      <th scope="col">surName</th>
+      <th scope="col">email</th>
+      <th scope="col">roomId</th>
+      <th scope="col">checkInDate</th>
+      <th scope="col">checkOutDate</th>
+      <th scope="col">numberOfNight</th>
+    </tr>
+  </thead>
+  <tbody>
+    {
+        results.map((item)=>{
+            return(
+                <tr>
+      <th scope="row">{item.id}</th>
+      <td>{item.title}</td>
+      <td>{item.firstName}</td>
+      <td>{item.sureName}</td>
+      <td>{item.email}</td>
+      <td>{item.roomId}</td>
+      <td>{item.checkInDate}</td>
+      <td>{item.checkOutDate}</td>
```

Comment:
this is good, your data is showing up as expected in the table & you've used moment to calculate the nights stayed, well done